openvdb/openvdb/math/Vec3.h

```diff
@@ -228,8 +229,8 @@ class Vec3: public Tuple<3, T>
     /// this = v1 cross v2, v1 and v2 must be distinct objects than "this"
     const Vec3<T>& cross(const Vec3<T> &v1, const Vec3<T> &v2)
     {
-        // assert(this!=&v1);
-        // assert(this!=&v2);
+        OPENVDB_ASSERT(this!=&v1);
+        OPENVDB_ASSERT(this!=&v2);
```

Comment:
I went through the ones I uncommented, as far as I can tell they are all valid asserts. I suspect they were commented out in these hot paths because of the perceived danger of NDEBUG being enabled